src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -3351,15 +3351,14 @@ bool clang_c_convertert::is_ConstructorOrDestructor(
          fd.getKind() == clang::Decl::CXXDestructor;
 }
 
-bool clang_c_convertert::perform_virtual_dispatch(
-  const clang::MemberExpr &member)
+bool clang_c_convertert::perform_virtual_dispatch(const clang::MemberExpr &)
 {
   // It just can't happen in C
   return false;
 }
 
 bool clang_c_convertert::is_fd_virtual_or_overriding(
-  const clang::FunctionDecl &fd)
+  const clang::FunctionDecl &)
```

Comment:
wouldn't this cause an unused argument warning?

Reply:
It would cause such warning if CC_DIAGNOSTIC_POP was properly defined instread of: 
```cpp
#define CC_DIAGNOSTIC_POP() _Pragma("GCC diagnostic push")
```

Reply:
Can you just remove the variable since it's not needed?

Reply:
Do you mean remove the entire thing `const clang::FunctionDecl &`?  But it's needed by `clang_cpp_convertert::is_fd_virtual_or_overriding`. 

Also, I think my previous reply might seem a bit confusing. 
> wouldn't this cause an unused argument warning?
`const clang::FunctionDecl &` wouldn't cause this warning. `const clang::FunctionDecl &fd` would have caused such warning if CC_DIAGNOSTIC_POP was properly defined. Both issues are fixed in this PR.